modules/integration-tests/src/main/java/org/apache/fluo/integration/TestUtil.java

```diff
@@ -27,17 +27,13 @@ private TestUtil() {}
   private static final Bytes ZERO = Bytes.of("0");
 
   public static void increment(TransactionBase tx, Bytes row, Column col, int val) {
-    int prev = 0;
     String prevStr = tx.get(row, col, ZERO).toString();
-    prev = Integer.parseInt(prevStr);
-    tx.set(row, col, Bytes.of(prev + val + ""));
+    tx.set(row, col, Bytes.of(prevStr + val));
```

Comment:
Your latest change forces it to be a string concatenation with `val`. I think it was actually supposed to be a sum first, and then converted to a string, as in `Bytes.of(Integer.toString(Integer.parseInt(prevStr) + val))`

Sorry, I feel like I've led you down a rabbit hole. If you just want to undo this last change, I think your previous changes were fine.

Reply:
Thanks for the feedback! I've updated the PR.